packages/amplify-util-mock/package.json

```diff
@@ -30,19 +30,19 @@
     "extract-api": "ts-node ../../scripts/extract-api.ts"
   },
   "dependencies": {
+    "@aws-amplify/amplify-appsync-simulator": "2.8.1-codegen-graphql15.0",
     "@hapi/topo": "^5.0.0",
-    "amplify-appsync-simulator": "^2.3.12",
-    "amplify-category-api-dynamodb-simulator": "2.4.4-beta.3",
-    "amplify-category-function": "4.2.0-beta.4",
-    "amplify-codegen": "^3.3.4",
-    "amplify-provider-awscloudformation": "7.0.0-beta.4",
-    "amplify-storage-simulator": "^1.7.0-beta.4",
+    "amplify-category-api-dynamodb-simulator": "2.4.4",
+    "amplify-category-function": "^4.2.1",
+    "amplify-codegen": "3.3.6",
+    "amplify-provider-awscloudformation": "7.0.0-a-cdkv2-graphql-15.1",
```

Comment:
I shipped `7.0.0-a-cdkv2-graphql-15.1` of this to mitigate test failures

This is replicating steps taken on main, i.e. here:
https://github.com/aws-amplify/amplify-category-api/blob/9a9b5e929de8126da2c6c058bbf93b79bf8db81c/packages/amplify-util-mock/package.json#L38